vendor/assets/javascripts/initializers.js

```diff
@@ -20,7 +23,7 @@ $(document).on('turbolinks:load', function() {
   $(".js-filter-search").each(function(e) {
     $(this).on('keyup', function(e) {
       if (e.keyCode === 13) {
-        $(this).closest('form').submit();
+        window.location.search  = "?utf8=%E2%9C%93&searched_term=" + $(this)[0].value;
```

Comment:
Oops, that's from the search for items within the collections view that got scrapped, I'll get rid of it.